core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -499,7 +500,16 @@ class Partition(val topicPartition: TopicPartition,
         addingReplicas = addingReplicas,
         removingReplicas = removingReplicas
       )
-      createLogIfNotExists(partitionState.isNew, isFutureReplica = false, highWatermarkCheckpoints)
+      try {
+        this.createLogIfNotExists(partitionState.isNew, isFutureReplica = false, highWatermarkCheckpoints)
+      } catch {
+        case e: ZooKeeperClientException =>
+          stateChangeLogger.info(s"Because a ZooKeeper client exception has occurred, completed become leader " +
+            s"state change from epoch $leaderEpoch only for those updated partitions with before " +
```

Comment:
The message seem inaccurate since we only skip this partition now. Ditto in makeFollower().

Reply:
That makes sense. That will be reflected. 👍